var/spack/repos/builtin/packages/wrf-hydro/package.py

```diff
@@ -0,0 +1,231 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import shutil
+import tempfile
+
+import llnl.util.tty as tty
+
+from spack import *
+
+
+class WrfHydro(Package):
+    """WRF-Hydro is a community modeling system and framework
+    for hydrologic modeling and model coupling.
+    """
+
+    homepage = "https://ral.ucar.edu/projects/wrf_hydro/overview"
+    git = "https://github.com/NCAR/wrf_hydro_nwm_public.git"
+    url = "https://github.com/NCAR/wrf_hydro_nwm_public/archive/refs/tags/v5.2.0.tar.gz"
+
+    maintainers = ["katrinaebennett", "ryanlcrumley", "daniellivingston"]
```

Comment:
Just to check, did these other users express a desire to be notified? FYI future PRs to this package will ping everyone in this list so I wanted to make sure.